addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -785,6 +786,19 @@ registerModel({
                 }
             }
         },
+        async _registerConnectionType(token, peerConnection) {
+            const rtcSession = this.messaging.models['RtcSession'].findFromIdentifyingData({ id: token });
```

Comment:
We need to start using models rather than strange stuff.

Start using models as soon as possible in flux (so do that line in the parent instead and pass the record as param).